sycl/include/CL/sycl/detail/builtins.hpp

```diff
@@ -248,8 +295,17 @@ MAKE_CALL_ARG3(select, __FUNC_PREFIX_OCL) // select
 #undef __NOEXC
 #undef MAKE_CALL_ARG1
 #undef MAKE_CALL_ARG2
+#undef MAKE_CALL_ARG2_SAME
 #undef MAKE_CALL_ARG3
 #undef PPCAT_NX
 #undef PPCAT
```

Comment:
Could you also add `__SYCL_` prefix for all other pre-processor functions as well, while you are here, please?

Reply:
Done